vpr/src/place/place.cpp

```diff
@@ -1252,11 +1328,17 @@ static float starting_t(const t_annealing_state* state, t_placer_costs* costs, t
     VTR_LOG("std_dev: %g, average cost: %g, starting temp: %g\n", std_dev, av, 20. * std_dev);
 #endif
 
-    /* Set the initial temperature to the standard of deviation divided by 64 */
-    /* so that the initial temperature adjusts according to the circuit */
-    /* and also keep the initial placement qaulity (not destroying it completely) */
-    /* and fine-tune the initial placement with the anneal*/
-    float init_temp = (std_dev / 64);
+    float init_temp = 0.0;
+
+    /*There were some issues with PR#2175 where the reduced initial
+     * temperature caused problems with the NoC placement. Current fix is
+     * to change the initial temperature back to its orignal value when
+     * NoC placement is turned on, otherwise use the reduced temperature.*/
+    if (noc_opts.noc) {
+        init_temp = 20. * std_dev;
```

Comment:
@Srivat97 Since this PR is mine, I would like to know what was the problem with starting temp lower than 20. * std_dev?
Does that mean that NoC placement quality was not okay since there weren't enough place time to find a good solution or did you face an error out (like seg fault)?

Reply:
Updated comment